module #2. REST API Basics/rest-api-basic/core/src/main/java/com/epam/esm/model/GiftCertificate.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.esm.model;
+
+import lombok.*;
+import org.apache.logging.log4j.core.config.plugins.validation.constraints.NotBlank;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+@EqualsAndHashCode
+public class GiftCertificate implements Serializable {
+    private static final long serialVersionUID = 7808945757372799936L;
```

Comment:
There is no need to implement Serializable and specify serialVersionUID. All data will read and saved into corresponding column in a table. It will not be any object serialization. It should be removed.